test/client/global.js

```diff
@@ -49,6 +57,17 @@ describe(`globals cases`, () => {
         }
     });
 
+    it('should get the default stage host equal to undefined', () => {
```

Comment:
What do you think about adding a `beforeEach()` to this describe block for resetting state on `window`?

Reply:
Move some declarations to the beforeEach and clean steps to the afterEach.